src/partials/options.html

```diff
@@ -225,7 +225,7 @@ <h5 class="section-heading">Coloring</h5>
     </div>
   </div>
 
-  <div ng-if="ctrl.panel.coloringType == 'thresholds'">
+  <div ng-if="ctrl.panel.coloringType == 'thresholds' || ctrl.panel.coloringType == 'pallete'">
```

Comment:
Can we use `enum` here instead of string?

Reply:
but how are we going to make that enum available in the template?

Reply:
didn't get any answer, so skipping this for now